src/register_axis.cpp

```diff
@@ -10,19 +10,22 @@
 #include <boost/histogram/python/register_axis.hpp>
 #include <vector>
 
-void register_axes_options(py::module &opt) {
-    opt.attr("none")      = (unsigned)bh::axis::option::none;
-    opt.attr("underflow") = (unsigned)bh::axis::option::underflow;
-    opt.attr("overflow")  = (unsigned)bh::axis::option::overflow;
-    opt.attr("circular")  = (unsigned)bh::axis::option::circular;
-    opt.attr("growth")    = (unsigned)bh::axis::option::growth;
-}
-
 void register_axes(py::module &ax) {
-    // This factory makes a class that can be used to create axes and also be used in
-    // is_instance
-    py::object factory_meta_py
-        = py::module::import("boost_histogram.utils").attr("FactoryMeta");
+    py::class_<options>(ax, "options")
+        .def_property_readonly("none", &options::none)
```

Comment:
No need to expose `none`, right?

Reply:
Yes, the options should be equal comparable.

Reply:
And pickleable (Is that a word?)

Reply:
Agreed. (I have no idea if it is, works for me though ;) )